docs/src/developing/programming-model/runtime.md

```diff
@@ -104,8 +104,9 @@ for more information.
 A transaction may request a specific level of `max_units` it is allowed to
 consume by including a
 [``ComputeBudgetInstruction`](https://github.com/solana-labs/solana/blob/db32549c00a1b5370fcaf128981ad3323bbd9570/sdk/src/compute_budget.rs#L39).
-Transaction prioritization depends on the fee/compute-unit ratio so transaction
-should request the minimum amount of compute units required for them to process.
+Transaction overall cost include prioritization-fee for every 10K compute-units,
+so transaction should request the minimum amount of compute units required
+for them to process.
```

Comment:
This description isn't as descriptive as it could be for something this critical

```
Note a transaction's cost to the payer is priced per 10k *requested* compute units, even if the actual compute units consumed during execution is far less than the *requested* amount. 

Thus, a transaction should request the minimum amount of compute units required for them to execute the transaction.
```